### DISCUSSION TITLE
OT: Saving image metadata in A1111

### DISCUSSION THREAD

**Discussion by Aamir3d** [MoonRide303/Fooocus-MRE#171] [General]
@MoonRide303 I saw the comment in Fooocus about A1111 and external metadata - 
If you enable this option in Settings > Saving images/grids , the metadata will be saved within the image and make it easier to use.

![image](https://github.com/MoonRide303/Fooocus-MRE/assets/122131887/f75ab8a6-39cd-44b6-8a1d-41b78108b175)

**Comment by MoonRide303** [comment]
Yeah, I've seen your comment - and wasn't aware that A1111 has that option. I will check it out.
